stance-4forums/data/fetchData.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+readonly DATA_URL='https://linqs-data.soe.ucsc.edu/public/psl-examples-data/stance.zip'
```

Comment:
Since we agreed on the "milti" approach to examples, break up this zip into two different ones.
Make the top-level directory "stance-4forums" for this one, and similar for the other example.